### DISCUSSION TITLE
Display 2.4 Zoll um 90! Grad verdreht.

### DISCUSSION THREAD

**Discussion by fraeggle** [beabel/radiowecker#59] [Fragen & Antworten / Q&A] [answered]
Ich habe hier ein 2.4 Zoll Touch (aus Alie.... und ja selber schuld..). Bei diesem haben die es tatsächlich hinbekommen, dass der Touch im gegensatz zum Bild, bei dem Radiowecker genau 90 GEGEN den Uhrzeigersinn verdreht ist.
Heißt beispielsweise, tippe ich oben Links wird die Lautstärke auf auf dem Startbildschirm auf Minimum gesetzt. Tippe ich UNTEN Links dann auf Maximal.
Kann man diese X/Y verschiebung auch anpassen?

**Comment by beabel** [comment, marked as answer]
Dann musst du diesen [Code ](https://github.com/beabel/radiowecker/discussions/53#discussioncomment-10347477)anpassen:

da du jedoch höchstwahrscheinlich rotation 3 verwendest musst du das aus rotation 1 zu 3 kopieren und je nachdem die Umrechnung anpassen.

**Reply by fraeggle** [reply]
Daaanke...  habs mit viiiiel Hirnschmalz hinbekommen........ Das blöde war tatsächlich die 90 Grad verschiebung.
180 wäre einfacher gewesen.

    case 3:
      pTransformed.x = (tft.height() - p.y) * 1.33;
      pTransformed.y = p.x * 0.75;

Zum Glück gabs mit dem Original 2.8 Zoll nicht so ein Chaos........  ;-)
